foxglove_bridge_base/src/test/test_client.cpp

```diff
@@ -12,43 +12,37 @@ namespace foxglove {
 constexpr auto DEFAULT_TIMEOUT = std::chrono::seconds(5);
 
 std::vector<uint8_t> connectClientAndReceiveMsg(const std::string& uri,
-                                                const std::string& topic_name) {
+                                                const std::string& topicName) {
   // Set up text message handler to resolve the promise when the topic is advertised
-  foxglove::Client<websocketpp::config::asio_client> wsClient;
+  auto wsClient = std::make_shared<foxglove::Client<websocketpp::config::asio_client>>();
```

Comment:
Why does this need to be a shared_ptr? Would unique_ptr work? I don't see the ownership being actually shared anywhere?

Reply:
You are right, will see what I can do. Can just pass a raw pointer to `waitForChannel` then